RELEASE-NOTES.txt

```diff
@@ -7,6 +7,7 @@ Unreleased
 * [**] Add support for setting heading anchors [#2947]
 * [**] Disable Unsupported Block Editor for Reusable blocks [#3067]
 * [**] Add proper handling for single use blocks such as the more block [#3042]
+* [**] Make inserter long-press options "add to beginning" and "add to end" always available. [#3074]
```

Comment:
👋 I realised that this change is assigned to `1.47` milestone but here in the release notes is under the `1.46.0` section, it would be nice to have it under the `Unreleased` section instead of here since it wasn't released on that version.

cc @AmandaRiu

Reply:
@fluiddot I'll open another PR to make that change 👍

Reply:
Thanks for the fix! I've just approved and merge it 🎊 .